src/runtime_src/core/common/query_requests.h

```diff
@@ -32,6 +32,26 @@ namespace xrt_core {
 
 namespace query {
 
+/**
+ * enum bdf - used for indexing the pcie bdf tuple (domain, bus, dev, func)
+ *
+ * pcie_bdf query request call returns the BDF in tuple format.
+ *   domain is at index 0
+ *   bus is at index 1
+ *   dev is at index 2
+ *   func is at index 3
+ * enum bdf member "max" is used for identifying tuple size.
+ * Retrieve the individual elements from bdf tuple using below enum bdf members.
+ */
+enum bdf
+{
+  domain = 0,
+  bus = 1,
+  dev = 2,
+  func = 3,
+  max = 4,
+};
```

Comment:
Plain enums are not allowed under any circumstances in C++.   I see the enum values used in some indexing 
 in string conversion in XBUtilities.cpp, but it doesn't accomplish anything when make_tuple is still used.

While I think a tuple of 4 entries for 'dbdf' is pretty darn obvious with a comment under struct query::pcie_bdf, I am okay if the tuple is replaced with an embedded struct under struct query::pcie_bdf, but the enum is not needed at all and must be removed.

Reply:
Removed enum code.